Sources/AccountSDKIOSWeb/User/User.swift

```diff
@@ -87,9 +87,9 @@ public class User: UserProtocol {
     
     /**
      Generate URL with embedded one-time code for creating a web session for the current user.
-
-     - parameter clientId: which client to get the code on behalf of, e.g. client id for associated web application
-     - parameter redirectURI: where to redirect the user after the session has been created
+     
+     - parameter clientId: which client to get the code on behalf of, e.g. client id for associated web application. Need to be from same merchant.
+     - parameter redirectURI: where to redirect the user after the session has been created. Has to match redirectURI in SelfService for client associated to clientId parameter.
```

Comment:
"The exact redirect URI must be registered for the given client in Self Service. " - sounds better. what do you think?

Reply:
Yes definitely! updating